documentation/specs/proposed/Wasm-tasks.md

```diff
@@ -0,0 +1,344 @@
+# Wasm/WASI tasks in MSBuild
+Exploration of using Wasm/WASI to create sandboxed [Tasks in MSBuild](https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-tasks) using non-dotnet Wasm/WASI compatible language.
```

Comment:
MS Learn guidelines are to remove the lang-culture from URLs. Then the user’s browser preferred language(s) are used.